Source/JavaScriptCore/runtime/JSPromise.cpp

```diff
@@ -219,13 +219,19 @@ void JSPromise::reject(JSGlobalObject* lexicalGlobalObject, JSValue value)
     }
 }
 
-void JSPromise::rejectAsHandled(JSGlobalObject* lexicalGlobalObject, JSValue value)
+// https://webidl.spec.whatwg.org/#mark-a-promise-as-handled
+void JSPromise::markAsHandled(JSGlobalObject* lexicalGlobalObject)
```

Comment:
I'm adding this in https://github.com/WebKit/WebKit/pull/28356/

Reply:
Nice, thanks! We'll still have to do some work in this PR since with the change in behaviour some of the navigation-api tests time out, but at least this makes the patch simpler and smaller.